src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -192,6 +192,7 @@ export class PatientViewPageStore {
 
     readonly hasMutationalSignatureData = remoteData({
         invoke: async() => false,

```

Comment:
seems odd. its `false` all the time

Reply:
Yes, we are temporary disable the feature until the backend is ready